Yoast/wordpress-seo
# Open graph article modified and publish time presenters and presentations

## Summary


This PR can be summarized in the following changelog entry:

* Adds open graph article modified and publish time presenters and presentations

## Relevant technical choices:

* I've renamed all occurrences of `publish time` to `published time` to make the naming consistent with the meta tag name (except for in the filter name, of course).
* I've introduced a get_post_type helper.

## Test instructions

This PR can be tested by following these steps:


* Install the classic editor plugin.
* Create a post in the classic editor.
* Go to the frontend. See the published time being output.
* Modify the post.
* Go to the frontend. See the published time and modified time being output.
* Create a post in Gutenberg.
* Go to the frontend. See the published time and modified time being output (because GB sets a modified time one or two seconds after the published time).
* Go to the home page. See no published or modified date being output.

## UI changes

## Documentation

## Quality assurance


Fixes #
